app/models/user.rb

```diff
@@ -1,2 +1,6 @@
-class User < ApplicationRecord
+class User < ApplicationRecordvalidates :name,  presence: true, length: { maximum: 50 }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: { maximum: 255 }, format: { with: VALID_EMAIL_REGEX }, uniqueness: { case_sensitive: false }
+  has_secure_password
+  validates :password, presence: true, length: { minimum: 6 }
```

Comment:
Remove space after the of curly brace and before ending brace
``
{minimum: 6}
``
And do it every where when you use curly brace